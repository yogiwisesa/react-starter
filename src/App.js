import React from 'react';

const App = props => (
  <div>
    <p>hello world</p>
    <img src="/dist/images/dog.jpg"/>
  </div>
)

export default App;