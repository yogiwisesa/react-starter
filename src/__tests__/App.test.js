import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

test('Should render App.js', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});
