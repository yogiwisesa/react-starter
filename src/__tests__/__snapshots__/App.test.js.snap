// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render App.js 1`] = `ShallowWrapper {}`;
